### DISCUSSION TITLE
backend repo int

### DISCUSSION THREAD

**Discussion by shaitanaleksander** [ratifire/devrate-service#3] [General]
Давайте тут обговоримо що потрібно нам в репозиторіі
- назва
- бібліотеки
- про що могли забути але потрібно щоб було

**Comment by MaksymMatveichuk** [comment]
Файлы конфигурации?

**Comment by ovoskresenskyy** [comment]
"Бібліотеки" то ти маєшь на увазі README ?

**Reply by shaitanaleksander** [reply]
spring boot mvc, lombok......

**Reply by ovoskresenskyy** [reply]
Зрозумів. Щось типу такого?
 - Spring Boot Starter Web
 - Spring Boot Starter Data JPA
 - Spring Boot Starter Test
 - Spring Boot Starter Validation ?
 - Postgresql
 - Lombok
 - JUnit Jupiter API

**Reply by MaksymMatveichuk** [reply]
Які бібліотеки потрібно теж тут будемо обговорювати?

Кеш будемо ставити? Може потрібно буде щось типу Redis?

**Reply by abalmasDA** [reply]
до списку вище я б додав ще:

- spring-boot-starter-security
- springfox-boot-starter
- mockito-core
- modelmapper

**Reply by IvashchenkoIvanIgorevich** [reply]
Гадаю треба ще щось для логування. По типу log4j

**Reply by ovoskresenskyy** [reply]
Log4j є як частина Lombok. Я тим користувався. Чи потрібно окремо під'єднувати залежність?

**Reply by shaitanaleksander** [reply]
в spiring boot вже є якесь логування https://docs.spring.io/spring-boot/docs/1.2.9.BUILD-SNAPSHOT/reference/html/howto-logging.html

**Reply by IvashchenkoIvanIgorevich** [reply]
Swagger , якщо звичайно його ще немає у залежностях spring.
Ще можливо треба бібліотека - jjwt (Java JWT) для токенів.

**Reply by abalmasDA** [reply]
щодо свагера, то я додав поінт springfox-boot-starter

**Comment by IvashchenkoIvanIgorevich** [comment]
Назва, це замість ratifire ?
Якщо так, то можна - devEval

**Reply by shaitanaleksander** [reply]
назва це назва репо

**Reply by IvashchenkoIvanIgorevich** [reply]
Тоді по аналогії з фронтенд -
devrate_back

**Reply by shaitanaleksander** [reply]
https://github.com/ratifire/devrate-service
ось це репо для бека поки, його не чипайте поки, потрібно налаштувати доступи, щоб будь хто не пушив в main гілку
